### ISSUE TITLE
Unbound Ailments Support gem not calculate correctly

### ISSUE THREAD

**Issue by pongsacha** [PathOfBuildingCommunity/PathOfBuilding#2148]
* **What happened?**
* trying to scale Shock 60% and Chill 40% with awake and non awake unbound ailment support gem but the number is not calculate for it. I am Elementalist so it can 100% Chill and Shock when hit with spell.
* 
* **What were you trying to do?**
the effect of elemental ailment can be calculate by cluster jewel that increase the effect but when put togather with Unbound ailment the number stay the same (its like no effect at all)
- 400% Shock Effect should result in 60% shock increase damage already but PoB still said im around 55-58% (with unbound)

* **What steps will reproduce the bug?**
try turn on/off the Unbound Ailments gem on Hydrosphere and unsocket the Right Medium Cluster jewel to see if the effect still the same or get higher 

* **Does it reproduce every time?**
yes

* **Provide a build code:**
https://pastebin.com/XBDwhiTY

TLDR : Unbound Ailments support is not working correctly with shock and chill effect in pob but the cluster jewel with increase effect work just fine

* **Screenshots:**
![image](https://user-images.githubusercontent.com/65549606/107303017-d5e0cd00-6ab0-11eb-943f-b8743fe61359.png)
![image](https://user-images.githubusercontent.com/65549606/107303053-e8f39d00-6ab0-11eb-8639-a1d592a6d555.png)
![image](https://user-images.githubusercontent.com/65549606/107303174-29ebb180-6ab1-11eb-895a-b187df0e5a9c.png)

**Comment by tympapost** [comment]
I'm having the same issue with my Elementalist.
Having Arc supported by Unbound elements and choosing that the enemy is shocked from the configuration tab does not affect the calculations of DPS. 
Other passives seem to be working fine with "enemy is shocked" selected.

**Comment by circlewind** [comment]
Same issue. It also exists for Scion with Elementalist path selected.

**Comment by Lilylicious** [comment]
Path of building only supports ailment effect in cases where a guaranteed shock is applied. The original issue in this thread is that Elementalist was not getting the benefits, which it should. As of tonight's release (2.20), it does. 

Scion is a different case. While Scion does have a minimum shock, it does not have the modifier that hits always shock. To be quite honest, I don't think Scion should get the 10% it currently does at all. Either way this is a separate issue that should get a separate issue opened.
